onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -91,17 +106,40 @@ def testSetProvidersWithOptions(self):
             option['trt_int8_calibration_table_name'] = calib_table_name
             int8_use_native_calibration_table = "true"
             option['trt_int8_use_native_calibration_table'] = int8_use_native_calibration_table
+            dla_enable = "true"
+            option['trt_dla_enable'] = dla_enable
+            option['trt_dla_core'] = new_dla_core
+            dump_subgraphs = "true"
+            option['trt_dump_subgraphs'] = dump_subgraphs
+            engine_cache_enable = "true"
+            option['trt_engine_cache_enable'] = engine_cache_enable
+            engine_cache_path = '/home/onnxruntime/engine_cache'
```

Comment:
why /home/onnxruntime ?
would this cause problems if you can't write to that location in the unit test?